src/openrct2-ui/windows/Changelog.cpp

```diff
@@ -41,273 +41,293 @@ static constexpr const rct_string_id WINDOW_TITLE = STR_CHANGELOG_TITLE;
 constexpr int32_t MIN_WW = 300;
 constexpr int32_t MIN_WH = 250;
 
-static rct_widget window_changelog_widgets[] = {
+static rct_widget s_wndChangelogWidgets[] = {
```

Comment:
We don’t use Hungarian notation. `_windowChangelogWidgets` would be more fitting I think. @IntelOrca ?

Reply:
I've made this change, do we want to make this change for the constexpr fields as well?